metrics/core/metrics.go

```diff
@@ -73,17 +73,6 @@ var LabeledMetrics = []Metric{
 	MetricFilesystemInodesFree,
 }
 
-var NodeAutoscalingMetrics = []Metric{
-	MetricNodeCpuCapacity,
```

Comment:
Let's leave metrics:
```
MetricNodeCpuCapacity,
MetricNodeMemoryCapacity,		
MetricNodeCpuAllocatable,		
MetricNodeMemoryAllocatable,
```
which are generally useful.

Let's also rename `NodeAutoscalingEnricher` to just `NodeEnricher`